coalib/parsing/inline_comments.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+
+def read_ignore_comments(original_line, start_comment, end_comment=''):
+    """
+    Read Ignore Comment(s) already present in the line and
```

Comment:
Comment(s) -> comment .

This algorithm has no support for multiple comments on the same line.

Reply:
``ignore_comments`` holds all the comments (regex matches) on the same line.